app/assets/javascripts/discourse/app/components/d-editor.js

```diff
@@ -439,6 +439,7 @@ export default Component.extend(TextareaTextManipulation, {
         const cookedElement = document.createElement("div");
         cookedElement.innerHTML = cooked;
 
+        // TODO (martin)  FIGURE
         linkSeenHashtags(cookedElement);
```

Comment:
This only happens `enable_diffhtml_preview` is true and I've never seen it before now, hopefully can just use the new lookup/linker for preview here